### PR TITLE
Add claud3 blueprint

### PR DESCRIPTION
**Pull request by Hailong-am**
### Description
Add claude3 blueprint. The request/response body is different with claude2, so i add a new file for claude3.

claude3
```
{
  "modelId": "anthropic.claude-3-sonnet-20240229-v1:0",
  "contentType": "application/json",
  "accept": "application/json",
  "body": {
    "anthropic_version": "bedrock-2023-05-31",
    "max_tokens": 1000,
    "messages": [
      {
        "role": "user",
        "content": [
          {
            "type": "image",
            "source": {
              "type": "base64",
              "media_type": "image/jpeg",
              "data": "iVBORw..."
            }
          },
          {
            "type": "text",
            "text": "What's in this image?"
          }
        ]
      }
    ]
  }
}
```

claude2
```
{
  "modelId": "anthropic.claude-v2:1",
  "contentType": "application/json",
  "accept": "*/*",
  "body": "{\"prompt\":\"\\n\\nHuman: Hello world\\n\\nAssistant:\",\"max_tokens_to_sample\":300,\"temperature\":0.5,\"top_k\":250,\"top_p\":1,\"stop_sequences\":[\"\\n\\nHuman:\"],\"anthropic_version\":\"bedrock-2023-05-31\"}"
}
```
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docs/remote_inference_blueprints/bedrock_connector_anthropic_claude3_blueprint.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        "anthropic_version": "bedrock-2023-05-31"
+    },
+    "actions": [
+        {
+            "action_type": "predict",
+            "method": "POST",
+            "headers": {
+                "content-type": "application/json"
+            },
+            "url": "https://bedrock-runtime.us-east-1.amazonaws.com/model/anthropic.claude-3-sonnet-20240229-v1:0/invoke",
```
Comment (ylwu-amzn) [inline docs/remote_inference_blueprints/bedrock_connector_anthropic_claude3_blueprint.md:49]:
```suggestion
            "url": "https://bedrock-runtime.${parameters.region}.amazonaws.com/model/${parameters.model}/invoke",
```

File: docs/remote_inference_blueprints/bedrock_connector_anthropic_claude3_blueprint.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        "secret_key": "<PLEASE ADD YOUR AWS SECRET KEY HERE>",
+        "session_token": "<PLEASE ADD YOUR AWS SECURITY TOKEN HERE>"
+    },
+    "parameters": {
+        "region": "<PLEASE ADD YOUR AWS REGION HERE>",
+        "service_name": "bedrock",
+        "auth": "Sig_V4",
+        "response_filter": "$.content[0].text",
+        "max_tokens_to_sample": "8000",
+        "anthropic_version": "bedrock-2023-05-31"
```
Comment (ylwu-amzn) [inline docs/remote_inference_blueprints/bedrock_connector_anthropic_claude3_blueprint.md:40]:
```suggestion
        "anthropic_version": "bedrock-2023-05-31",
        "model": "anthropic.claude-3-sonnet-20240229-v1:0"
```

File: docs/remote_inference_blueprints/bedrock_connector_anthropic_claude3_blueprint.md (hunk 1 above)
Comment (ylwu-amzn) [reply docs/remote_inference_blueprints/bedrock_connector_anthropic_claude3_blueprint.md:49]: Do same change for AWS Opensearch Service blueprint below

File: docs/remote_inference_blueprints/bedrock_connector_anthropic_claude3_blueprint.md (hunk 2 above)
Comment (ylwu-amzn) [reply docs/remote_inference_blueprints/bedrock_connector_anthropic_claude3_blueprint.md:40]: Do same change for AWS Opensearch Service blueprint below

**Review by jngz-es — APPROVED** [review]

**Comment by Hailong-am** [comment]
@jngz-es @ylwu-amzn seems that backport is not triggered, do we need to backport this to 2.x?

**Pull request merged**